common/test/acceptance/tests/helpers.py

```diff
@@ -279,23 +279,25 @@ def setUp(self):
         self.event_collection = MongoClient()["test"]["events"]
         self.reset_event_tracking()
 
-    def assert_event_emitted_num_times(self, event_name, event_time, event_user_id, num_times_emitted):
+    def assert_event_emitted_num_times(self, event_name, event_time, event_user_id, num_times_emitted, **kwargs):
         """
         Tests the number of times a particular event was emitted.
+
+        Extra kwargs get passed to the mongo query.
```

Comment:
This describe doesn't 100% describe what the kwargs are used for in this test. We should probably mention that we prefix them with `event.` before handing them off to Mongo.